test/style/blocks_test.exs

```diff
@@ -117,6 +117,23 @@ defmodule Styler.Style.BlocksTest do
       """)
     end
 
+    test "removes identity else clauses" do
+      assert_style(
+        """
+        with :ok <- b(), :ok <- b() do
+          :ok
+        else
+          :what -> :what
```

Comment:
unsure if i want to replace all equivalent nodes, or only variables. letting people specify patterns here at least tells the reader "the caller doesn't have to worry about anything but this value", whereas `else error -> error end` conveys 0 information and so can be trivially removed.